mathesar/models/query.py

```diff
@@ -210,6 +212,7 @@ def initial_columns_described(self):
 
     def _describe_query_column(self, sa_col):
         alias = sa_col.name
+        assert not alias.startswith('%')
```

Comment:
What's going on here? Could you add a comment?  It seems like this would affect the names available to the front end for aliases. Is that documented somewhere?

Reply:
Oh, that's a debugging leftover.